tests/simulator/test_initialization.py

```
@@ -691,15 +658,15 @@ def _test_domainparticles_have_correct_split_from_coarser_particle(
                     part2 = coarse_split_particles[pop_name].select(patch.box)
                     self.assertEqual(part1, part2)
 
-    def _test_patch_ghost_on_refined_level_case(self, dim, has_patch_ghost, **kwargs):
+    def _test_patch_ghost_on_refined_level_case(self, ndim, has_patch_ghost, **kwargs):
         import pyphare.pharein as ph
 
         out = "phare_outputs"
```

Ruff

Local variable `out` is assigned to but never used

Remove assignment to unused variable `out`

(F841)